osg-system-profiler

```diff
@@ -223,6 +276,17 @@ if $rpm_install; then
     dump_files_in_dir "/etc/osg/config.d"
 fi
 
+# XRootD cconfig dumps for available instances
+if command -v cconfig &>/dev/null; then
+    for file in /etc/xrootd/xrootd-*.cfg
```

Comment:
One minor nitpick (?): this grabs config based on what's on disk not necessarily what we expect so if a site is doing some wild stuff with their config locally, this will show up as an absence of the config, which may be unclear to the person doing the troubleshooting.

That being said, that sort of thing will show up in the `rpm -q --verify` and probably oddities in other areas.

Reply:
If a site's doing that kind of wild stuff with their config locally, they're going to have a hard time starting xrootd -- the systemd service file looks for the config in `/etc/xrootd/xrootd-%i.cfg` (where `%i` is the instance name)

Reply:
So essentially we query all the configs that _can_ be started with the systemd service files we ship.

Reply:
But yes this only works for installations controlled by systemd, i.e. non-container ones.